<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupTab</class>
 <widget class="QWidget" name="SetupTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QGroupBox" name="tcpIpGBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Tcp/IP</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="topMargin">
         <number>23</number>
        </property>
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="text">
           <string>Connect</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/root/images/connected.png</normaloff>:/root/images/connected.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectBtn">
          <property name="text">
           <string>Disconnect</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/root/images/disconnected.png</normaloff>:/root/images/disconnected.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="IpAddressLbl">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&amp;IP Address:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>ipAddressEditLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ipAddressEditLine"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="portLbl">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;&amp;Port:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>portEditLine</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEditLine">
          <property name="maximumSize">
           <size>
            <width>75</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="directCommandsGBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Direct Commands</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="commandsCBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="text">
           <string>Send</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/root/images/send.png</normaloff>:/root/images/send.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>13</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="receiveBtn">
          <property name="text">
           <string>Receive</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/root/images/receive.png</normaloff>:/root/images/receive.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Instrumemt Messages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QTextEdit" name="instrumentMessagesTxtEditor"/>
        </item>
        <item alignment="Qt::AlignTop">
         <widget class="QPushButton" name="clearBtn">
          <property name="text">
           <string>Clear</string>
          </property>
          <property name="icon">
           <iconset resource="../resources.qrc">
            <normaloff>:/root/images/clear.png</normaloff>:/root/images/clear.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>connectBtn</tabstop>
  <tabstop>disconnectBtn</tabstop>
  <tabstop>ipAddressEditLine</tabstop>
  <tabstop>portEditLine</tabstop>
  <tabstop>commandsCBox</tabstop>
  <tabstop>sendBtn</tabstop>
  <tabstop>receiveBtn</tabstop>
  <tabstop>instrumentMessagesTxtEditor</tabstop>
  <tabstop>clearBtn</tabstop>
 </tabstops>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
